src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/util/SalesforceSplitUtil.java

```diff
@@ -106,8 +107,8 @@ private static BatchInfo[] runBulkQuery(BulkConnection bulkConnection, String qu
     throws AsyncApiException, IOException {
 
     SObjectDescriptor sObjectDescriptor = SObjectDescriptor.fromQuery(query);
-    JobInfo job = SalesforceBulkUtil.createJob(bulkConnection, sObjectDescriptor.getName(),
-            getOperationEnum(operation), null);
+    JobInfo job = SalesforceBulkUtil.createJob(bulkConnection, sObjectDescriptor.getName(), getOperationEnum(operation),
+                                               null, ConcurrencyMode.Parallel);
```

Comment:
What is the use case here? Should the concurrency mode be passed here from the config ?

Reply:
The Salesforce Batch Source as well as Multi-source plugins also use the bulk job API and call the same `createJob()` method. 
No changes have been made as there aren't any lock contention issues observed in the Source plugins.